optaplanner-docs/src/modules/ROOT/pages/planner-configuration/planner-configuration.adoc

```diff
@@ -1793,6 +1793,49 @@ of the measuring.
 longest-running currently active solver.
 * `optaplanner.solver.solve.duration.seconds-duration-sum`: the sum of each active solver's solve duration. For example, if there are two active solvers, one running for three minutes and the other for one minute, the total solve time is four minutes.
```

Comment:
Haven't we changed the name of this metric (and the ones above) to not include the `-`  sign?

Reply:
It a bit complicated; `optaplanner.solver.solve.duration` is the metric/meter/LongTaskTimer, `seconds-duration-sum` is the measurement. Measurements don't really have standardized names